### ISSUE TITLE
Python tiledbsoma: assertion when creating zero-length SparseNdArray

### ISSUE THREAD

**Issue by bkmartinjr** [single-cell-data/TileDB-SOMA#297]
Python tiledbsoma.SparseNdArray will not create an array with shape (0,0).  It is often convenient to create these as placeholders.  It currently asserts when requested to do so.

---

Digging into this more (see below), currently electing to raise a usable exception, and not support (0,) shaped dimensions.

**Comment by johnkerl** [comment]
https://github.com/single-cell-data/TileDB-SOMA/pull/295

**Comment by bkmartinjr** [comment]
@johnkerl - I believe this is fixed and can be closed.

**Comment by bkmartinjr** [comment]
This no longer generates an AssertionError, but fails in another way:

```
In [1]: import tiledbsoma as soma
i
In [2]: import pyarrow as pa

In [3]: a = soma.SOMASparseNdArray("tmp/foobar")

In [4]: a.create(type=pa.float32(), shape=(0,0))

In [5]: a.shape
Out[5]: (2, 2)
```

**Comment by bkmartinjr** [comment]
Fixed by #356
